universal-login-react/src/core/utils/getPriceInEther.ts

```diff
@@ -0,0 +1,12 @@
+import {utils} from 'ethers';
+import {WeiPerEther} from 'ethers/constants';
+import {GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER} from '../constants/normalization';
+
+const {parseUnits} = utils;
+
+export const getPriceInEther = (priceInFiat: string, etherPriceInFiat: string): utils.BigNumber => {
+  const etherPriceNormalizedToInteger = parseUnits(etherPriceInFiat, GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER);
```

Comment:
Inline GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER or create private( no export) const in this file
Rename  GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER => DECIMAL_FOR_NORMALIZE